firefox-ios/firefox-ios-tests/Tests/XCUITests/DataManagementTests.swift

```diff
@@ -5,35 +5,54 @@
 import XCTest
 
 class DataManagementTests: BaseTestCase {
+    func cleanAllData() {
+        navigator.goto(WebsiteDataSettings)
+        mozWaitForElementToExist(app.tables.otherElements["Website Data"])
+        navigator.performAction(Action.AcceptClearAllWebsiteData)
+        XCTAssertEqual(app.cells.buttons.images.count, 0, "The Website data has not cleared correctly")
+        // Navigate back to the browser
+        mozWaitElementHittable(element: app.buttons["Data Management"], timeout: TIMEOUT)
+        app.buttons["Data Management"].tap()
+        app.buttons["Settings"].waitAndTap()
+        app.buttons["Done"].waitAndTap()
+    }
+
     // Testing the search bar, and clear website data option
     // https://mozilla.testrail.io/index.php?/cases/view/2307015
     func testWebSiteDataOptions() {
+        navigator.openURL("www.youtube.com")
```

Comment:
Would the test still work if we use local web sites instead?

Reply:
Updated with local links. If is ok please resolve the conversation